### PR TITLE
Components

### PR DESCRIPTION
**Pull request by astroboyReloaded**
## For this activity, I did the following: 📑 
- Remove unused files. ➖
- Create `components` folder and `calculator.js` file. ➕
- Add basic **style**. ✏️ 
😎

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <button type="button" className="key">
+        .
+      </button>
+      <button type="button" className="key key-operator">
+        =
+      </button>
+    </div>
+  </div>
+);
+
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] Kindly ensure your calculator has at least one child component.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @astroboyReloaded,

## Highlights
✔️ Functional calculator component

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (astroboyReloaded) [reply src/components/Calculator.js]: Ok, understood! Thank you!!

**Review by levy002 — APPROVED** [review]
## Hello @astroboyReloaded 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Well-detailed PR descriptions and title :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by astroboyReloaded** [comment]
Thank you!

**Pull request merged**